### DISCUSSION TITLE
How to set a default selection set for a model?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [prisma/prisma#11497] [Q&A] [answered]
Can I set a default selection set for a specific model? I've created a validator for a model and I don't want to pass it to every query. can I use middleware for that?

**Comment by amirhhashemi** [comment, marked as answer]
I found a very popular open issue about that (https://github.com/prisma/prisma/issues/5042) so I think it's not implemented yet.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
